### DISCUSSION TITLE
Autocomplete is intermittently not working

### DISCUSSION THREAD

**Discussion by bisbaldk** [microsoft/vscode-cpptools#6975] [Q&A] [answered]
MacBook Pro 2018 Big Sur 11.1

VS Code Version: 1.53.2

extension version: 1.2.1

I've been facing this problem for weeks, literally can't find solution to it,

autocompletion works, but not all the time, usually it works for like 20 or 30minutes, then the vscode got stuck,
no autocompletion, the error squiggles also won't disappear even the error is corrected,
had to command + shift + p to reload the whole window, then autocomplete is back, and it works for another 5 minutes and got stuck somehow, and reload and got stuck and reload, over and over again...
Is there a permanent fix to this?

Edit: Just downgraded to version 1.1.3 waiting for the new patch to fix it.

**Comment by ChiSaw** [comment]
May be related with https://github.com/microsoft/vscode-cpptools/issues/6958 ?
A temporary downgrade to extension version 1.1.3 (and disabling auto-update extension for the time being) helped me while waiting for a fix.

**Reply by bisbaldk** [reply]
Thank you for this! I'd try it out and see what happens!

**Comment by sean-mcmanus** [comment, marked as answer]
Is 1.2.1 any better? My guess is that it's not. However, we've found/fixed an issue that could potentially cause this, so we're looking into release it as part of 1.2.2-insiders.

**Reply by bisbaldk** [reply]
I'm currently on 1.2.1, I believe this is the version that causes trouble unless 1.2.1 was released just now or yesterday, how long should I expect to be waiting until the release of 1.2.2?

**Reply by sean-mcmanus** [reply]
Can you check if https://github.com/microsoft/vscode-cpptools/releases/tag/1.2.2-insiders fixes the issue for you?

**Reply by bisbaldk** [reply]
Will definitely try this either later tonight or tomorrow, thanks for the work!

**Reply by bisbaldk** [reply]
> Can you check if https://github.com/microsoft/vscode-cpptools/releases/tag/1.2.2-insiders fixes the issue for you?

Using 1.2.2 now, it seems to work fine for now, I'll report here if I further encounter any problem, thanks.
